src/app.ts

```diff
@@ -0,0 +1,90 @@
+import { Probot, Context } from 'probot';
+
+interface PullRequestInfo {
+  owner: string
+  repo: string
+  pull_number: number
+}
+
+const getPRInfo = async (context: Context, {
+  owner, repo, pull_number
+}: PullRequestInfo) => {
+  const {
+      data: {
+        commits,
+        head
+      }
+    } = await context.octokit.pulls.get({
+      owner, repo, pull_number
+    });
+  
+  return { prNumCommits: commits, prHeadSha: head.sha, prBranchName: head.ref }
+}
+
+const createEmptyCommit = async (
+  context: Context,
+  {
+    owner,
+    repo,
+    prHeadSha
+  }: PullRequestInfo & { prHeadSha: string }
+) => {
+  const { data: { sha } } = await context.octokit.git.createCommit({
+        owner,
+        repo,
+        message: 'empty commit',
```

Comment:
Should we consider this commit message configurable (via `squash-commit-app.yml`) for example? If you agree I will open an issue where we can discuss this further.

Reply:
let worry about configuration later. For now, I'd set a more descriptive commit message, something like this
```suggestion
        message: `empty commit to preset the squash & merge commit subject from the pull request title
        
Created by https://github.com/gr2m/squash-commit-app`,
```